packages/client/src/components/CategoriesListDropDown/CategoriesListDropDown.component.js

```diff
@@ -16,9 +16,9 @@ const DropDownView = ({
   };
   const optionList =
     options.length > 0 &&
-    options.map((item, index) => {
+    options.map((item) => {
       return (
-        <option key={index.id} value={item}>
+        <option key={Math.random()} value={item}>
```

Comment:
I think you've misunderstood what `key` does. 

`key` is used by the framework to recycle DOM elements so it has to be **consistent**. Using random here is like pouring coca cola into your car engine...
The issue is that `index` is a number, so `index.id` doesn't exist. Just put back `index`.

Reply:
If I can remove console error by replacing index but its showing error while rendering in browser. Do not use Array index in keys. https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md

Reply:
Its remove one error by key={item.toString()} and 2nd one remove error by key={product.id} because I found product.id in array.